lib/rubocop/cop/betterment/not_using_rswag.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Betterment
+      class NotUsingRswag < Base
+        MSG = 'API tests should use documented using rswag and not the built in `get`, `post`, `put`, `patch`, `delete` methods'
```

Comment:
How does it distinguish between API tests and request tests that test ERB endpoints?

Reply:
As written it won't distinguish between the two, I hadn't considered that case.

I think if we wanted to write spec for an endpoint that produces html we'd just specify `produces 'text/html` and not specify anything about the structure of it. In the tests themselves whether using rswag's built in `run_test!` or our Retail helper `submit_request!` we will still have access to the `response` and could assert things about the html from that

This approach is perhaps a bit too naive, instead we could try to assert that a request spec is written in a particular structure using methods that come from rswag:

```ruby
  path 'url_string' do
    ...
    get/post/put/patch/delete 'some_description' do
      ...
      response 'http_status_code', 'some_description' do
        ...
        run_test!
        
        or

        it 'some_description' do
          submit_request!
        end
      end
    end
  end
```